src/requestHandlers/hook.ts

```diff
@@ -269,6 +269,30 @@ const _validateHookConnectionController = (sequelize : Sequelize) => async(next_
     }
     return { success: true };
 };
+
+const getAllHooksByPanelSetId = async (req: Request, res: Response): Promise<Response> => {
+    const id = req.params.id;
+    const response = await _getAllHooksByPanelSetIdController(sequelize)(Number(id));
```

Comment:
Should validate arguments like in other request handlers
```suggestion
    const id = Number(req.params.id);
    const validArgs = assertArgumentsNumber({ id });
    if (!validArgs.success) return res.status(400).json(validArgs);
```